Source/RunActivity/Viewer3D/Processes/GameStateRunActivity.cs

```diff
@@ -104,6 +104,11 @@ internal override void Update(RenderFrame frame, double totalRealSeconds)
 
         internal override void Load()
         {
+            // The virtual file system must be initialized before loading anything.
+            Vfs.LogLevel = Game.Settings.VfsLogLevel;
+            Vfs.NoAutoMount = Game.Settings.VfsNoAutoMount;
+            Vfs.Initialize(Game.Settings.Menu_Selection[0], Path.GetDirectoryName(System.Windows.Forms.Application.ExecutablePath));
```

Comment:
This will fail if you run `RunActivity` directly, as developers and tester do. You **must** pass all information needed to load the correct content via the command line.